stan/math/rev/core/var.hpp

```diff
@@ -408,6 +416,15 @@ class var_value {
     return var_value<double>(vari_coeff);
   }
 
+  /**
+   * View element of eigen matrices
+   * @param i Row to access
+   * @param j Column to access
+   */
+  inline auto coeffRef(Eigen::Index i, Eigen::Index j) const {
+    return this->coeff(i, j);
+  }
+
```

Comment:
I only added these because they are needed by the `matrix<var>` version in elt_multiply

Reply:
Is it more efficient to do this than `variable.adj().coeffRef(...)`? I've just been doing that since I think it's the right thing for `var<mat>`.

Reply:
It's less efficient, it's just that this method needs to be defined for the `var<matrix>` version that does the loop. I wouldn't mind waiting until #2064 goes in for this because there we actually do all of these much more nicely